packages/vm/src/evm/opcodes/EIP2929.ts

```diff
@@ -18,7 +18,7 @@ export function accessAddressEIP2929(
   address: Address,
   common: Common,
   chargeGas = true,
-  isSelfdestruct = false
+  isSelfdestructOrAuthCall = false
```

Comment:
should this be `isSelfdestructOrAuthcall`? we should ensure we are using the same case style for every use.